src/role_harvester.js

```diff
@@ -24,7 +24,7 @@ roles.harvester.settings = {
     1: [2, 1, 1],
     3: {
       0: [2, 1, 1],
-      400: [2, 2, 2],
+      400: [1, 1, 1]
```

Comment:
The case is `energyAvailable = 600`. With the change it builds `[3, 3, 3]` creeps, before it would only do `[2, 2, 2]` ones. I think the lowest possible combination makes the most sense.